### PR TITLE
SLVSCODE-953 untar eslint-bridge bundle and send its path to slls

### PR DESCRIPTION
**Pull request by sophio-japharidze-sonarsource**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -200,6 +200,7 @@ export async function activate(context: VSCode.ExtensionContext) {
           omnisharpDirectory: Path.resolve(context.extensionPath, 'omnisharp'),
           csharpOssPath: Path.resolve(context.extensionPath, 'analyzers', 'sonarcsharp.jar'),
           csharpEnterprisePath: Path.resolve(context.extensionPath, 'analyzers', 'csharpenterprise.jar'),
+          eslintBridgeServerPath: Path.resolve(context.extensionPath, 'eslint-bridge')
```
Comment (jblievremont) [inline src/extension.ts:203]: See my comment on SLLS: the `additionalAttributes` field is sent as-is to the telemetry, so I believe it needs to be renamed and cleaned-up.

**Review by jblievremont — CHANGES_REQUESTED** [review]
Looks good on the functional side (and tested manually); let's make sure that we are not sending anything unnecessary in the telemetry payload 😅

**Review by jblievremont — APPROVED** [review]
LGTM ⏩!

**Comment by sophio-japharidze-sonarsource** [comment]
Superseded by #661

**Pull request closed without merging**